test/SlowTests/Issues/RavenDB_21263.cs

```diff
@@ -0,0 +1,334 @@
+
+        private const string ExceptionMessage = "Revision change vector cannot exceed 1536 bytes";
+
+        [RavenTheory(RavenTestCategory.Smuggler | RavenTestCategory.BackupExportImport)]
```

Comment:
Please make test that 

1. Import dump file (using smuggler):
```
var operation = await store.Smuggler.ForDatabase(store.Database)
                            .ImportAsync(new DatabaseSmugglerImportOptions(), ms);
```

2. One dump file should not have a `SupportedFeatures` field
Second dump should have `SupportedFeatures` empty or null (think which one is valid case)

3. after you import such file assert that we add the current `SupportedFeatures` to the new database

Reply:
done